l10n_es_sigaus_sale/models/sale_order_line.py

```diff
@@ -40,3 +40,15 @@ def _compute_invoice_status(self):
             else:
                 line.invoice_status = "no"
         return super(SaleOrderLine, self - sigaus_lines)._compute_invoice_status()
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        lines = super().create(vals_list)
+        if not self.env.context.get("avoid_line_recursion"):
+            sales = self.env["sale.order"]
+            for line in lines.filtered(lambda li: li.product_id.sigaus_has_amount):
+                sales += line.order_id
+            for sale in sales:
+                sale.automatic_sigaus_exception()
```

Comment:
This function could be implemented in multiple ways, not just api.one. Can you review it?
The call can be made much better if the change is made.

Reply:
I can, i'll do the improvement in a day or two

Reply:
you will have to check which other sites are affected.